solutions/USACO open16-closing-the-farm.mdx

```diff
@@ -170,4 +174,70 @@ void dfs(int barn)
 
 </CPPSection>
 
+<JavaSection>
+
+```java
```

Comment:
use tabs instead of spaces to indent

Reply:
Fixed this just now, can you check if it's right?

Reply:
Yep, it looks good now.